src/styles/container.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  display: grid;
+  grid-template-areas: "first"
+    "second"
+    "third"
+    "fourth"
+    "fifth"
+    "sixth";
+
+  &__block {
+    display: flex;
+    min-width: 300px;
+    height: 300px;
+    color: white;
+    justify-content: center;
+    align-items: center;
+  }
+
+  @each $index, $color in $colors {
+    & :nth-child(#{$index}) {
+      background-color: $color;
+    }
+  }
+
+  @each $index, $number in $numbers {
+    & :nth-child(#{$index}) {
+      grid-area: $number;
+    }
+  }
+}
+
+@media (min-width: 600px) and (max-width: 900px) {
+  .container {

```

Comment:
I think and (max-width: 900px) here is redundant, because of your next media query. Correct me, someone, if I'm wrong.

Reply:
Ihor is right